keras_nlp/src/utils/transformers_model_utils/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 The KerasNLP Authors
```

Comment:
Maybe let's rename this to `keras_nlp/src/utils/transformers/`? Clear enough, and avoids long dir names.

Reply:
Also I don't think we need the hf prefix everywhere. Something like this? Feel free to tweak.

```python
src/utils/transformers/convert.py # Move the transformer_utils into this new dir?
src/utils/transformers/convert_gemma.py
src/utils/transformers/convert_llama3.py
src/utils/transformers/safetensor_utils.py
```